db/db_test2.cc

```diff
@@ -2422,6 +2444,7 @@ TEST_F(DBTest2, TestPerfContextIterCpuTime) {
 
   SetPerfLevel(PerfLevel::kEnableTimeAndCPUTimeExceptForMutex);
   iter = db_->NewIterator(ReadOptions());
+  ASSERT_OK(iter->status());
```

Comment:
Move to after the last iter->Valid() check

Reply:
Done